modules/web/server/src/main/resources/app.conf

```diff
@@ -46,7 +49,7 @@ web-server {
 # Configuration of the observe engine
 observe-engine {
     # host for the odb
-    odb = "ws://localhost:8082/ws"
+    odb = "wss://lucuma-postgres-odb-dev.herokuapp.com/ws"
```

Comment:
Can we also allow an override for this so we can use a local ODB? I guess an env variable will do.

Reply:
I wonder how. with an env variable?